src/app/Home.py

```diff
@@ -1,6 +1,8 @@
 import streamlit as st
 
-# Read and display the content from the Markdown file
-with open("home_markdown.md", "r") as f:
-    content = f.read()
-st.markdown(content)
+st.sidebar.image("Conservation X Labs CXL logo.png", use_column_width=True)
+
+from common.render_method import render_markdown
+
+# st.sidebar.image('Conservation X Labs CXL logo.png', use_column_width=True)
+render_markdown("home_markdown.md")
```

Comment:
This will probably need to be `src/app/home_markdown`. See this PR: https://github.com/uw-ssec/neglected-diagnostics/pull/62

when testing, please test from the root:
```
streamlit run src/app/Home.py
```
This ensures you are using the same working directory as the docker container does when deployed. See `deployment/Dockerfile`

Reply:
Same comment applies to the other calls to `render_markdown`

Reply:
Made the changes